python_files/data_preprocessor.py

```
@@ -0,0 +1,180 @@
+            'Source_City',
+            'Device_Name',
+            'KYC_Status',
+            'Channel',
+            'ID_Dest',
+            'Dest_State',
+            'Dest_City']
+                
+
+    encoder = OrdinalEncoder(handle_unknown='use_encoded_value', unknown_value=-1)
+
+    le_data = pd.DataFrame(
```

Ruff (0.8.2)

Local variable `le_data` is assigned to but never used

Remove assignment to unused variable `le_data`

(F841)